### ISSUE TITLE
[TensorRT] Unsupported operation _ResizeNearest_TRT / _PyramidROIAlign_TRT /_CropAndResize...

### ISSUE THREAD

**Issue by tingggggg** [NVIDIA/TensorRT#685]
## Description

I build the env refer https://github.com/NVIDIA/TensorRT/issues/123#issuecomment-551269792.
And I success got the maskRCNN&fasterRCNN .uff file use sample code, then use trtexec to do inference all got PASSED.

But when I want to use own .py code to build trt engine with above succefull .uff file got the following ERROR: UFFParser

[TensorRT] ERROR: UffParser: Validator error: roi_pooling_conv_1/CropAndResize_new: Unsupported operation _CropAndResize
[TensorRT] ERROR: UffParser: Validator error: roi_align_mask_trt: Unsupported operation _PyramidROIAlign_TRT

I already do the 'ctype.CDLL("/usr/src/tensorrt/lib/libnvinfer_plugin.so")' in my .py code.

Why trtexec can succeed, but my code is not.
Did i miss something?

## Environment

**TensorRT Version**: 6.0.1
**GPU Type**: 
**Nvidia Driver Version**: 
**CUDA Version**: 10.1
**CUDNN Version**: 
**Operating System + Version**: 
**Python Version (if applicable)**: 
**TensorFlow Version (if applicable)**: 
**PyTorch Version (if applicable)**: 
**Baremetal or Container (if container which image + tag)**:

**Comment by tingggggg** [comment]
update info
------ 2020.07.23 ------ 
After use `trt.init_libnvinfer_plugins(trt.Logger(trt.Logger.WARNING), "")` in my .py code
Then use 
 ```
PLUGIN_CREATORS = trt.get_plugin_registry().plugin_creator_list
for i in PLUGIN_CREATORS:
        print(i.name)
```
can got the custom plugins (like above)

> GroupNormalizationPlugin
CustomEmbLayerNormPluginDynamic
CustomFCPluginDynamic
CustomGeluPluginDynamic
CustomQKVToContextPluginDynamic
CustomSkipLayerNormPluginDynamic
GridAnchor_TRT
GridAnchorRect_TRT
NMS_TRT
Reorg_TRT
Region_TRT
PriorBox_TRT
Normalize_TRT
RPROI_TRT
BatchedNMS_TRT
FlattenConcat_TRT
CropAndResize
DetectionLayer_TRT
Proposal
ProposalLayer_TRT
PyramidROIAlign_TRT
ResizeNearest_TRT
SpecialSlice_TRT
InstanceNormalization_TRT
GenerateDetection_TRT
MultilevelProposeROI_TRT
MultilevelCropAndResize_TRT
CoordConvAC

But use trt.Build engine with uff still got the same error

> [TensorRT] ERROR: UffParser: Validator error: roi_pooling_conv_1/CropAndResize_new: Unsupported operation _CropAndResize

Why CropAndResize this plugin still can not be parse successful, any step that I miss it ?

Please~~

**Comment by tingggggg** [comment]
Finally, I found need to init the plugin first.
`trt.init_libnvinfer_plugins(trt.Logger(trt.Logger.WARNING), "")`
